### PR TITLE
impl-ed order and item in order

### PR DESCRIPTION
**Pull request by SalnikovMarkiian**

### CODE REVIEW

File: src/main/java/com/example/onlinebookstore/service/order/OrderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    @Override
+    public OrderItemDto getOrderItemById(Authentication authentication, Long orderId, Long id) {
+        OrderItem orderItem = orderItemRepository
+                .findByIdAndOrderAndUser(id, orderId, authentication.getName())
+                .orElseThrow(() ->
+                        new EntityNotFoundException(
+                                "Item with id " + id + " does not exist in order with "
+                                        + orderId + " id")
+                );
```
Comment (viacheslav-torbin) [inline src/main/java/com/example/onlinebookstore/service/order/OrderServiceImpl.java:75]: This reverse pyramid of code looks kinda sus
Use String.format

File: src/main/java/com/example/onlinebookstore/dto/order/CreateOrderRequestDto.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.example.onlinebookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.Length;
+
+public record CreateOrderRequestDto(
+        @NotNull @Length(max = 128)
```
Comment (liliia-ponomarenko) [inline src/main/java/com/example/onlinebookstore/dto/order/CreateOrderRequestDto.java]: better use NotBlank with Strings

File: src/main/resources/db/changelog/changes/10-create-orders-table.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_orders_users
+                    references: users(id)
+              - column:
+                  name: status
+                  type: varchar(16)
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/changes/10-create-orders-table.yaml:25]: Is it enough?

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -16,4 +16,8 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/08-create-shopping-carts-table.yaml
   - include:
-      file: db/changelog/changes/09-create-cart-items-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/09-create-cart-items-table.yaml
+  - include:
+      file: db/changelog/changes/10-create-orders-table.yaml
+  - include:
+      file: db/changelog/changes/11-create-order-items-table.yaml
```
Comment (liliia-ponomarenko) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: add one empty line

**Review by liliia-ponomarenko — APPROVED** [review]
Great 😊  A few recommendations ;)

File: src/main/resources/db/changelog/changes/10-create-orders-table.yaml (hunk 3 above)
Comment (SalnikovMarkiian) [reply src/main/resources/db/changelog/changes/10-create-orders-table.yaml:25]: i think so)

**Pull request merged**